### ISSUE TITLE
Recompiling GPU software for A100

### ISSUE THREAD

**Issue by mboisson** [ComputeCanada/software-stack#95]
This is the list of toolchains available: 

- [x] CUDAcore,11.0.2 => CUDAcore,11.4.2
- [x] gcccorecuda,2020a => gcccorecuda,2020.1.114
- [x] gcccuda,2020a => gcccuda,2020.1.114
- [x] iccifortcuda,2020a => iccifortcuda,2020.1.114
- [x] gmklc,2020a => gccflexiblascuda,2020.1.114
- [x] gompic,2020a => gompic,2020.1.403.114
- [x] iompic,2020a => iompic,2020.1.403.114
- [ ] gomklc,2020a => gofbfc,2020.1.403.114 (with FFTW)
- [x] gomklc,2020a => gofbc,2020.1.403.114. (without FFTW)
- [x] imklc,2020a => iccifortflexiblascuda,2020.1.114
- [ ] iomklc,2020a => iofbfc,2020.1.403.114 (with FFTW)
- [x] iomklc,2020a => iofbc,2020.1.403.114. (without FFTW)
- [ ] nvhpccuda,2020a => nvhpccuda,2020.1.114
- [x] fosscuda,2020a => gofbc,2020.1.403.114
- [x] nvompic => ?

This is the list of most used software packages that depend on cuda/11.0, with their associated toolchains. 
- [x] libfabric (gcccorecuda)
- [x] openmpi (iccifortcuda, gcccuda)
- [x] openmpi (nvhpccuda)
- [x] ucx (gcccorecuda)
- [x] gdrcopy (gcccorecuda)
- [x] cudnn (CUDAcore)
- [ ] fsl (gmklc) (fails to build)
- [x] arrow (gcccuda)
- [x] nccl (CUDAcore)
- [x] opencv (gmklc => gccflexiblascuda, iimklc=>iccifortflexiblascuda)
- [x] fftw-mpi (gompic, iompic)
- [x] boost-mpi (gompic, iompic)
- [x] amber (gomklc => gofbc). (done)
- [x] gromacs-plumed (gomklc) -- (Oliver: blocked by plumed)
- [x] plumed (gomklc => gofb, gofbc) (done)
- [x] gromacs (gomklc)
- [x] mpi4py (gompic/iompic)
- [x] namd-multicore (iimklc => iccifortcuda + FFTW dependency)
- [x] openmm (gomklc, iomklc)
- [x] ambertools (fosscuda => gofbc) (done)
- [x] scalapack (gompic) (done)
- [x] namd-ofi-smp (iimklc)
- [x] hdf5-mpi (nvompic)
- [x] cgns (nvompic)
- [x] vmd (iimklc => iccifortflexiblascuda) (done)
- [x] namd-ucx-smp (iimklc)
- [x] arrayfire (gomklc) (in progress.... succeeds to build, fails to run with segfault)
- [x] faiss (gmklc => gccflexiblascuda)
- [x] openmm-alphafold (gomklc)
- [x] itk (gcccuda)
- [x] opa-psm2 (gcccorecuda)
- [x] vasp (iomklc)
- [x] magma (gmklc => gccflexiblascuda)

**Comment by mboisson** [comment]
Closing with won't fix for the other ones.
